hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -1050,6 +1050,14 @@ private HddsProtos.DatanodeUsageInfoProto getUsageInfoFromDatanodeDetails(
       DatanodeDetails node, int clientVersion) {
```

Comment:
The implementation of this method largely seems redundant now that some other additions have been made. NIT but we could replace its implementation with something like this:
```
    int containerCount = -1;
    try {
      containerCount = scm.getScmNodeManager().getContainers(node).size();
    } catch (NodeNotFoundException ex) {
      LOG.error("Received container report from unknown datanode {}.",
              node, ex);
    }
    DatanodeUsageInfo usageInfo = scm.getScmNodeManager().getUsageInfo(node);
    usageInfo.setContainerCount(containerCount);
    return usageInfo.toProto(clientVersion);
```